build/linux/installer/scripts/dcr-config-parser.rb

```diff
@@ -0,0 +1,53 @@
+#!/usr/local/bin/ruby
+
+require 'fileutils'
+require 'json'
+require_relative 'ConfigParseErrorLogger'
+
+@os_type = ENV['OS_TYPE']
+@controllerType = ENV['CONTROLLER_TYPE']
+@containerType = ENV['CONTAINER_TYPE']
+@daemonset = 'daemonset'
```

Comment:
if only reference once, might not need global var

Reply:
which one in specific are you referring?

Reply:
daemonset one